### PR TITLE
New common run config vertxWebEnvironment

### LINKED ISSUE #142: Option for running vertx web in dev mode
Vert.x Web has a dev mode that should be enabled by default and possibly turned off via config.

### PR DESCRIPTION
**Pull request by tsegismont**
Closes #142

### CODE REVIEW

**Comment by tsegismont** [comment]
@cescoffier @kameshsampath can you please review? Thanks!

File: src/main/asciidoc/inc/_common_run-configuration.adoc [hunk 1]
```diff
@@ -15,14 +15,22 @@ If a yaml file is configured then it will be converted to json by the plugin.  T
...
+
 | workDirectory
 | The working directory of the running process of the application
 | vertx.directory
 | `${project.basedir}`
+
+| vertxWebEnvironment
+| The link:https://vertx.io/docs/vertx-web/java/#_development_mode[environment] the Vert.x Web app is running in.
+| vertxweb.environment
+| Default to either the `VERTXWEB_ENVIRONMENT` environment variable value or, if absent, to `dev`.
```
Comment (cescoffier) [inline src/main/asciidoc/inc/_common_run-configuration.adoc]: Should it be `VERTX_WEB_ENV` (or `ENVIRONMENT`)

Comment (tsegismont) [reply src/main/asciidoc/inc/_common_run-configuration.adoc]: No, it should be the same as defined by Vert.x web https://vertx.io/docs/vertx-web/java/#_development_mode

Comment (cescoffier) [reply src/main/asciidoc/inc/_common_run-configuration.adoc]: Ok make sense.

**Pull request merged**